docs/Training-On-Your-Data.md

```diff
@@ -4,13 +4,13 @@ This guide is meant to be a detailed walkthrough of training a model on your own
 
 The basic steps are:
 
-- [ ] [Configure your environment/cloud](#environment-setup)
```

Comment:
Trying to fix the bad rendering on the website:
![Captura de pantalla 2025-03-09 a la(s) 20 16 59](https://github.com/user-attachments/assets/74b1a54e-d095-4b00-8361-3999f2d12c93)